_includes/extra_styles/highlight.css

```diff
@@ -0,0 +1,26 @@
+.highlight .hll { background-color: #404040; }
+.highlight { border-radius: 4px; background: #202020; color: #d0d0d0; }
+.highlight :is(.c, .ch, .cm, .cpf, .c1) { color: #ababab; font-style: italic; }
+.highlight .err { color: #a61717; background-color: #e3d2d2; }
+.highlight :is(.esc, .g, .l, .n, .o, .x, .p, .ld, .ni, .nl, .nx, .py, .pm) { color: #d0d0d0; }
+.highlight :is(.k, .kc, .kd, .kn, .kr, .kt, .nt, .ow) { color: #6ebf26; font-weight: bold; }
+.highlight .cp { color: #cd2828; font-weight: bold; }
```

Comment:
i think #cd2828 is a little too dark to parse easily on the dark background; perhaps #e74242?

Reply:
It's only used for "preprocessor directives" and it's bold, I think it's fine:
![image](https://user-images.githubusercontent.com/1322658/206885185-f11b80bf-4657-474c-ba5b-51fab3fe47b2.png)

Reply:
you're right. For some reason it was rendering quite differently on my browser earlier (but now appears as above?). review marked approved.